home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/query/EntitySpecificationService.java

```diff
@@ -15,12 +15,15 @@
 
     public static final String RSQL_EQUAL = "==";
 
+    public static final String RSQL_EQUAL_OR = "=in=";
+
     public static final String RSQL_AND = ";";
 
     private static String toRSQLString(Map<String, String> filter) {
         return filter.entrySet().stream()
             .filter(entry -> entry.getValue() != null)
-            .map(entry -> entry.getKey() + RSQL_EQUAL + entry.getValue())
+            .map(entry ->
+            entry.getKey() + RSQL_EQUAL_OR + "(" + entry.getValue() + "," + entry.getValue().toUpperCase() + ")")
```

Comment:
I think we should find a better way to fix it.

Please consider narrowing the query parameter to `$ref: '#/components/schemas/ContactType'` and converting the value appropriately.